Tests/SyncTests/CryptoTests.swift

```diff
@@ -53,4 +53,26 @@ c2l0cyI6W3siZGF0ZSI6MTMxOTE0OTAxMjM3MjQyNSwidHlwZSI6MX1dfQ==
     func testBadBase64() {
         XCTAssertNil(Bytes.decodeBase64(invalidB64))
     }
+
+    func testBase64DecodeUrlSafe() {
```

Comment:
Added some tests for the Base64Decode conversion to ensure we're still keeping the proper functionality